src/ui/libs/DatalensChartkit/components/Control/Items/Items.js

```diff
@@ -238,10 +238,16 @@ function BaseControlTextArea({label, theme, value, placeholder, onChange}) {
     }, [value, text]);
 
     const buttonTheme = theme in legoThemeNameMapper ? legoThemeNameMapper[theme] : theme;
+    const buttonSize = isMobileView ? MOBILE_SIZE.BUTTON : 's';
 
     return (
         <React.Fragment>
-            <Button view={buttonTheme || 'normal'} width="max" onClick={() => setShowModal(true)}>
+            <Button
+                view={buttonTheme || 'normal'}
+                size={buttonSize}
+                width="max"
+                onClick={() => setShowModal(true)}
```

Comment:
Let's add `handleShowModal` instead of unanimous function.
General  recommendation is less unanimous functions. Little help for performance.